site/en/docs/extensions/reference/scripting/index.md

```diff
@@ -33,45 +33,49 @@ The only required field is `tabId`. By default, an injection will run in the
 main frame of the specified tab.
 
 ```js
-const tabId = getTabId();
-chrome.scripting.executeScript(
-    {
-      target: {tabId: tabId},
-      files: ['script.js'],
-    },
-    () => { ... });
+function getTabId() { ... }
+
+chrome.scripting
+  .executeScript({
+    target: {tabId: getTabId()},
+    files: ['script.js'],
+  })
```

Comment:
Throughout this CL, I'd recommend using google JS style / clang --js style.  Otherwise, we'll continue to have inconsistent styling.

Reply:
Ack. Preparing another PR to add some guidance to the dcc handbook about this. I'll also update this PR to follow this recommendation.